src/vsphere_cpi/spec/unit/cloud/vsphere/network_spec.rb

```diff
@@ -319,22 +327,25 @@ module VSphereCloud
                .with('switch-id').and_return(logical_switch)
             expect{
               network.destroy('switch-id')
-            }.to raise_error('Expected switch switch-id to have only one port. Got 0')
+            }.to raise_error
           end
         end
 
         context 'if more than 1 attached' do
-          let(:switch_ports) { [instance_double(NSXT::LogicalPort), instance_double(NSXT::LogicalPort)] }
+          let(:switch_ports) { [logical_port, logical_port] }
           it 'raises an error' do
             expect(router_provider).to receive(:get_attached_router_ids)
               .with('switch-id').and_return(['t1-router-id'])
             expect(switch_provider).to receive(:get_attached_switch_ports)
               .with('switch-id').and_return(switch_ports)
             expect(switch_provider).to receive(:get_switch_by_id)
              .with('switch-id').and_return(logical_switch)
+
+            expect(switch_provider).to receive(:get_logical_port_status)
+             .twice.and_return("UP")
             expect{
               network.destroy('switch-id')
-            }.to raise_error('Expected switch switch-id to have only one port. Got 2')
+            }.to raise_error
```

Comment:
using raise error without specifying a specific error or error message could make this test pass even when something else raises an error. So, it is best to put the error message back here

Reply:
Done